data/human/post-war reactions.txt

```diff
@@ -0,0 +1,260 @@
+				`	"Don't you see it was what we needed? The Pug made us set aside our pride and resolve our issues directly without continuing the conflict."`
+				`	"Oh, the Pug were real alright. I met some and fought them."`
+					goto nobody
+				`	"I think you both need to shave."`
+					goto scowl
+			`	One twin rolls his eyes while the other twitches his mustache awkwardly. He rewinds the video to the Parliamentary discussion, where Danforth is reassuring councilors about the Navy's ability to patrol the fringes of human space.`
+			`	The left twin says, "Look at this bozo! Bounty's Councilor Maduena Kabiru, he keeps ranting against this Navy guy, uh, this Admiral..." He snaps his fingers trying to remember the name.`
+			`	"Admiral Danforth," says the right twin.`
+			`	"Right. Danforth. This Councilor wants Danforth to defend against 'unknown dangers at the fringes of human space'. Don't we have enough immediate problems without imagining new horrors?"`
+			`	"Like terrorists nuking us," says the right twin.`
+			choice
+				`	"The Navy is skilled and honorable. I trust them to protect us."`
+					goto eyes
+				`	"No, don't worry about that. I ended the bomb threat."`
+				`	"Maybe you're right."`
+					goto shrug
+				`	"All we can do is get on with our lives, right?"`
+					goto shrug
+				`	"If you don't shave soon, people will think you're wearing masks."`
+					goto scowl
+			label nobody
+			`	Both of them give hard belly laughs, sending waves through their mustaches from the breeze of their breath.`
+			`	The left twin says, "Who are you, Admiral Danforts?" His twin corrects him. "Danforth. Admiral Danforth. Did the Navy fire you or are you here on holiday?"`
+			choice
+				`	"Have you been under a rock the whole war? I'm Captain <first> <last>!"`
+				`	"Never mind."`
+					goto shrug
+				`	"At least I don't have a disheveled hamster stuck to my upper lip."`
+					goto scowl
+			`	Both twins look confused, and the left one responds. "We've been mining asteroids the whole war. The war was a big deal, but most of us had to keep living our lives. Do you think we have time to keep track of every warring captain?"`
+			`	The right one says apologetically, "Thank you for your service, Captain <last>."`
+			`	They both give clumsy but well-intentioned salutes and return to their ships.`
+				decline
+			label eyes
+			`	The left twin laughs and says, "Pardon us if we keep our eyes open and trust to our own guns." Both twins nod and depart to their ships.`
+				decline
+			label shrug
+			`	Both twins shrug and leave for their ships.`
+				decline
+			label scowl
+			`	Both twins' faces twist in a way that might indicate embarrassment or annoyance. It's hard to tell since their facial hair is halting any emotive power. They walk away in the direction of their ships.`
+				decline
+
+
+mission "War Reactions: Navy and Free Worlds Captains"
+	landing
```

Comment:
`You are relaxing in a cafe [...]` doesn't really sound like the beginning of a `landing` mission.
```suggestion
```

Reply:
No, it doesn't sound that way to me either, but there are a lot of Hai Reveal landing missions that are written like spaceport missions. I'll apply your change, but I'll leave this unresolved for further discussion.